tools/wt_verify/wt_verify.py

```diff
@@ -153,7 +155,76 @@ def parse_chkpt_info(f):
     return [root_addr, line, chkpt_info]
 
 
-def parse_output():
+def parse_dump_blocks():
```

Comment:
I don't understand why we need to use the output from dump_blocks and then parse the data, when the same information is available in the output of dump_pages. Why not simply filter the information we need from the dump_pages output? 

That would save us all this code, or have I misunderstood something?

Reply:
As discussed, we could reuse that code but I wanted to isolate my code from the rest given the skunkworks context.